File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    let answer = 0;
    let tokens = binomial.trim().split(/\s+/);
    if(tokens[1] == "+"){
        return Number(tokens[0]) + Number(tokens[2]);
    } else if(tokens[1] == "-"){
        return tokens[0] - tokens[2];
    } else{
        return tokens[0] * tokens[2];
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        String[] tokens = binomial.trim().split("\\s+");
        int x = Integer.parseInt(tokens[0]);
        int y = Integer.parseInt(tokens[2]);
        
        if(tokens[1].equals("+")){
            return x + y;
        } else if(tokens[1].equals("-")){
            return x - y;
        } else{
            return x * y;
        }
    }
}